### ISSUE TITLE
improve accessibility: use correct html tags(<a>) for links 

### ISSUE THREAD

**Issue by hitchhooker** [polkassembly/polkassembly#1492]
seems like UI links are currently lacking <a></a> tags leading poor accessibility. 
highly recommend using google's pagespeed or inbuilt lighthouse in chromium(same thing)
https://pagespeed.web.dev/analysis/https-polkadot-polkassembly-io-opengov/hrboxhuqb8?form_factor=mobile 

i for example am using computer completely without mouse so those <a> tags in the websites are quite important for navigating around the website. 
j
![image](https://github.com/polkassembly/polkassembly/assets/15621959/51f34565-e97c-467f-8d31-81fb1de82cd8)
![image](https://github.com/polkassembly/polkassembly/assets/15621959/4524819c-4d55-40de-9293-10079420f96d)
here example of using just <h4> and lacking <a>

**Comment by Swapnil-Sagar** [comment]
Hi,

Thanks for pointing this out. We'll check for appropriate tags to improve accessibility. 
Appreciate your feedback!

Best,
Polkassembly Team
